app/src/main/res/layout/offer_header.xml

```diff
@@ -7,10 +7,11 @@
     android:paddingBottom="@dimen/base_margin_quadruple"
     tools:context=".feature.offer.ui.OfferActivity">
 
-    <com.hedvig.app.ui.view.HedvigCardView
+    <androidx.cardview.widget.CardView
```

Comment:
```suggestion
    <com.google.android.material.card.MaterialCardView
```

end-tag needs to be replaced as well.

Reply:
Both start and closing tags are androidx.cardview.widget.CardView :)